xbmc/platform/darwin/osx/XBMCApplication.mm

```diff
@@ -97,11 +104,28 @@ - (void)applicationWillFinishLaunching:(NSNotification*)notification
                                                        keyEquivalent:@"t"];
   floatOnTopMenuItem.keyEquivalentModifierMask = NSEventModifierFlagCommand;
   [windowMenu addItem:floatOnTopMenuItem];
+
+  NSMenuItem* hideMenuItem = [[NSMenuItem alloc] initWithTitle:@"Hide"
+                                                        action:@selector(hideAppAction:)
+                                                 keyEquivalent:@"h"];
+  hideMenuItem.keyEquivalentModifierMask = NSEventModifierFlagCommand;
+  [windowMenu addItem:hideMenuItem];
```

Comment:
normally Hide is under the \<app name\> menu (the first one, `appMenu` in code I guess)